From 6dcb5b7944bb1fa2510e354db2165a1943591ce7 Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Thu, 29 May 2014 17:22:03 +0800
Subject: [PATCH 4255/5965] m6tvc: correct mute gpio with GPIOAO_10 for h15

request by kieth.liu@amlogic.com
verified by haibo.you@amlogic.com
---
 arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd b/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
index c074800ca8d7..6c0ad82a8ba8 100755
--- a/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
@@ -528,7 +528,7 @@
 		aml,sound_card = "AML-M6TV";
 		//aml,codec_dai = "tas5711-","rt5616-aif1","rt5631-hifi","wm8960-hifi";
 		dev_name = "aml_m6tv_audio.0";
-		mute_gpio = "GPIOW_19";
+		mute_gpio = "GPIOAO_10";
 		pinctrl-names = "aml_m6tv_audio";
 		pinctrl-0 = <&aml_i2s_pins>;
 	};
-- 
2.19.0

